libcore/src/IO/IniFileParser.cpp

```diff
@@ -235,6 +235,10 @@ bool IniFileParser::ParseHeader(TiXmlNode * xHeader)
         xHeader->FirstChildElement("trajectories")->Attribute("fps", &fps);
         _config->SetFps(fps);
 
+        int precision = 2;
+        xHeader->FirstChildElement("trajectories")->Attribute("precision", &precision);
```

Comment:
When reading in a valid precision range should be checked

Reply:
See
c80537d6917da36f3145c35aaf9adf36008395e0